### ISSUE TITLE
[FE] Beta Program [Proof of Concept] 

### ISSUE THREAD

**Issue by Justin-Pickett** [department-of-veterans-affairs/va.gov-team#2747]
## Background

This is a first pass at looking to implement a "beta program enrollment" for Profile/Account combination work for the Personalization 2.0 initiative. 

## Goal
To create a proof of concept/ first pass of how the beta program will flow. This is being done to see if we actually will implement this along side the delivery of the Account/Profile combination work. 

## Assumptions
1. We can actually implement this while trying to do research on users. Lisa will provide more insights on any limitations of what type of research/info we can gather from our beta test users. 
2. 

## How to implement this 
follow these steps:

1. Create the content/Markdown page in `content/beta-enrollment` where you would like the enrollment button to reside. [Example from Personalization UAT](https://github.com/department-of-veterans-affairs/vets-website/blob/b2baab7e9ad26c7d464c417366f9241a3bbab660/content/pages/beta-enrollment/personalization.md).
2. Make sure to include a React root element in the Markdown file, `<div id="react-root"></div>`.
3. Register the route as React entrypoint by creating the button in `beta-enrollment/routes.js`.
4. Any applications that are part of that Beta should be wrapped in a `BetaApp` tags. For example, Personalization products were wrapped in:
    - `<BetaApp featureName={features.dashboard} redirect="/beta-enrollment/personalization/">`
    - In the event the user navigated there without being enrolled, they would be redirected to the enrollment page.
    - The FE and API have to agree on the name of your beta. Make sure it exists in the [API](https://github.com/department-of-veterans-affairs/vets-api/blob/cb0b37ec83fd5c46e7d8c918bae35027e8541696/config/routes.rb#L243) `beta_registration` controller.

## Unenrolling from Beta
There is an action available for deleting a service from the user's services array, but it must be implemented manually. [Here's](https://github.com/department-of-veterans-affairs/vets-website/blob/4caa3f5689de0767c85c30e260e9eda9fb953130/src/applications/personalization/account-beta/containers/BetaTools.jsx) how we did in the Personalization rollout. This `BetaTools` component was rendered into the Account application in the  [`AccountMain`](https://github.com/department-of-veterans-affairs/vets-website/blob/4caa3f5689de0767c85c30e260e9eda9fb953130/src/applications/personalization/account-beta/components/AccountMain.jsx#L78) component so that the user could easily unenroll from the Personalization beta features.

## Acceptance Criteria 

As a logged-in user to VA.gov
I want to be able to enroll into the beta program in the profile section
So that I can acess new features released to beta users for a specified amount of time.

Scenario: System user signs in with valid credentials

“Given I’m a authenticated user with LOA3 credentials 
and I’m on the Profile page

When I fill in the “Username” and “Password” fields with my authentication credentials
and I click the Sign-In button
Then the system prompts me to enroll into the beta program”

## Definition of Done
- [ ] This has been reviewed and approved by Depo Leadership.
- [ ] This is ready to be implemented in parallel with the release of combine Profile/Account work.

**Comment by Samara-Strauss** [comment]
Closing in favor of #10476
